.gitignore

```diff
@@ -1,2 +1,3 @@
 **/.idea
 **/.DS_Store
+**/packages/**
```

Comment:
seems a little odd to ignore packages here

Reply:
"I added this line following the suggestion from @xunliu . Perhaps it would be better to only include init/jupyter/packages instead? Let me know if that works."

Reply:
@FANNG1 Because we doesn't needs to commit these download jar files to git repo.